README.md

```diff
@@ -1188,6 +1191,43 @@ All options from BDD tests do also apply here.
 
 **NOTE:** Performance tests should be run without concurrency.
 
+#### List of test tags
+
+| Tag name           | Tag meaning                                                                        |
+| ------------------ | ---------------------------------------------------------------------------------- |
+| @smoke             | Smoke tests verifying basic functionality                                          |
+| @performance       | Performance tests                                                                  |
+| @olm               | OLM integration tests                                                              |
+| @travelagency      | Travel agency tests                                                                |
+|                    |                                                                                    |
+| @disabled          | Disabled tests, usually with comment describing reasons                            |
+| @cli               | Tests to be executed only using cli                                                |
+|                    |                                                                                    |
+| @springboot        | SpringBoot tests                                                                   |
+| @quarkus           | Quarkus tests                                                                      |
+|                    |                                                                                    |
+| @dataindex         | Tests including DataIndex                                                          |
+| @jobsservice       | Tests including Jobs service                                                       |
+| @managementconsole | Tests including Management console                                                 |
+| @infra             | Tests checking KogitoInfra functionality                                           | (should mark also every test which use Kogito infra?)
```

Comment:
Maybe @infra tag is too much vague? If a test is using Infinispan, we can use @infinispan. If it's also using @externalcomponent, I know that it's not being deployed using the Kogito infra.

Reply:
Well, it doesn't have to be so in future.
Right now when Kogito infra is used then we know that Infinispan/Kafka will be installed. But in the future some other providers can be used.

Reply:
we should have one tag for each component and one that could group all the dependent components (kafka, infinispan)
`@infra` was put at the beginning but we could change it to something like `dependency` ?

On those `@infra` we are testing that the installation of the dependencies is happening well.
So I would not annotate others that are using those components as it is not exactly what we are testing IMO ...

Reply:
> @infra was put at the beginning but we could change it to something like dependency ?

I would keep the `@infra` there as it is aligned to KogitoInfra object name.

> On those @infra we are testing that the installation of the dependencies is happening well.

ok